docs/digger/installation/run-digger-installer.asciidoc

```diff
@@ -26,23 +26,21 @@ ansible-playbook -i <your-inventory-sample> sample-build-playbook.yml
 +
 There are three points during the installation where there are manual steps required.
```

Comment:
There are now two points where the user needs to accept the licence. When initially installing AndroidSDK and then when we sync the Android SDK versions

Reply:
`. Carry out manual steps during the installation
+
There are two points in the installation where you have to manually accept Android SDK licence.
+
.. Initial Android SDK installation
.. Sync of Android SDK versions
+`

Reply:
resolved